packages/electron/src/client/main.js

```diff
@@ -16,6 +16,11 @@ const { schema } = require('../config/main')
 Event.__type = 'electronnodejs'
 
 module.exports = (opts) => {
+  // Sanity check api key has been provided
```

Comment:
Might be worth noting why we need to do this here but don't in other notifiers, e.g.:

```suggestion
  // check api key has been provided as we need it to create the FileStore
  // which happens before the API key is validated
```